src/Tests/SettingsDataTrait.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\acquia_lift\Tests\SettingsDataTrait.
+ */
+
+namespace Drupal\acquia_lift\Tests;
```

Comment:
Seems like this is duplicated for unit tests.  Is there a way to prevent the duplication?